### PR TITLE
Omit the `RULES` tab if the policy rules feature is inactive

### PR DESCRIPTION
**Pull request by mertyildiran**
This PR omits the `RULES` tab if the policy rules feature is inactive.

`--traffic-validation-file` option is **not** supplied:

Screenshot from 2021-09-24 14-42-16

`--traffic-validation-file` option is supplied:

Screenshot from 2021-09-24 14-42-52

Screenshot from 2021-09-24 14-42-57

Screenshot from 2021-09-24 20-13-03

Screenshot from 2021-09-24 20-14-59

### CODE REVIEW

File: ui/src/components/EntryDetailed.tsx [hunk 1]
```diff
@@ -62,6 +62,7 @@ const EntrySummary: React.FC<any> = ({data}) => {
 };
 
 export const EntryDetailed: React.FC<EntryDetailedProps> = ({entryData}) => {
+    console.log("entryData.rulesMatched:", entryData.rulesMatched)
```
Comment (gadotroee) [inline ui/src/components/EntryDetailed.tsx:65]: I guess this is a debug log that should be removed?
@mertyildiran

**Review by gadotroee — COMMENTED** [review]
after removing the debug log I think this is a good change

File: ui/src/components/EntryDetailed.tsx (hunk 1 above)
Comment (mertyildiran) [reply ui/src/components/EntryDetailed.tsx]: @gadotroee I'm sorry, I forgot to remove that.

**Review by gadotroee — APPROVED** [review]
Good change.

**Pull request merged**